modules/gapi/src/api/kernels_core.cpp

```diff
@@ -383,5 +383,40 @@ GMat warpAffine(const GMat& src, const Mat& M, const Size& dsize, int flags,
     return core::GWarpAffine::on(src, M, dsize, flags, borderMode, borderValue);
 }
 
+core::GDetections parseSSD(const GMat& in,
+                           const GOpaque<Size>& in_sz,
+                           const float& confidence_threshold,
+                           const int& filter_label)
```

Comment:
Don't see "alignment to square" flag among the parameters.

Reply:
Added `alignment to square` flag to kernel.